evm/its.js

```diff
@@ -437,6 +456,18 @@ async function processCommand(_, chain, options) {
             break;
         }
 
+        case 'decodeMulticall': {
+            const { multicallData } = options;
+
+            validateParameters({ isValidCalldata: { multicallData } });
+
+            const decodedMulticall = await decodeMulticallData(multicallData, IInterchainTokenService);
```

Comment:
can you add an example to the README? And example output to PR description for how it looks?

Reply:
yup